### ISSUE TITLE
After Install

### ISSUE THREAD

**Issue by robotrono** [np1/mps-youtube#234]
I get this error:
```
Traceback (most recent call last):
  File "/usr/local/bin/mpsyt", line 7, in <module>
    from mps_youtube import main
  File "/usr/local/lib/python3.2/dist-packages/mps_youtube/__init__.py", line 3,
 in <module>
    from .main import init
  File "/usr/local/lib/python3.2/dist-packages/mps_youtube/main.py", line 1436
    seek = u"[{0}\u2190{1}] seek [{0}\u2192{1}]"
                                               ^
SyntaxError: invalid syntax
```

Ubuntu 12.04 / VPS / Python 3.2

**Comment by eliottwiener** [comment]
Python 3.2 does not support the explicit unicode string literal. Support for such literals was restored in Python 3.3 (see [PEP 414](https://www.python.org/dev/peps/pep-0414/)). You can `from __future__ import unicode_literals` to fix the issue under Python 3.2.

**Comment by robotrono** [comment]
Thanks, how can play a YT Playlist with url?

mpsyt playurl https://www.youtube.com/playlist?list=PLbbZzFSu3a-i-xc5VARuamD-Ds11-KXTm 

Dont work
